src/main/java/org/opensearch/timeseries/TimeSeriesAnalyticsPlugin.java

```diff
@@ -1672,6 +1679,17 @@ public List<NamedXContentRegistry.Entry> getNamedXContent() {
             );
     }
 
+    @Override
+    public Collection<SystemIndexDescriptor> getSystemIndexDescriptors(Settings settings) {
+        List<SystemIndexDescriptor> systemIndexDescriptors = new ArrayList<>();
+        systemIndexDescriptors.add(new SystemIndexDescriptor(ALL_AD_RESULTS_INDEX_PATTERN, "Time Series Analytics Results index pattern"));
+        systemIndexDescriptors
+            .add(new SystemIndexDescriptor(ALL_AD_DETECTOR_INDEX_PATTERN, "Time Series Analytics Detector index pattern"));
+        systemIndexDescriptors.add(new SystemIndexDescriptor(CHECKPOINT_INDEX_NAME, "Time Series Analytics Checkpoints index"));
```

Comment:
Can you change description to "AD Checkpoints index"?

Reply:
Changed